Package@swift-5.6.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:5.6
```

Comment:
The style tool requires `swift-tools-version:5.6`. We can't modify the existing `Package.swift` file since that will break the build for old toolchains, so instead we can add a separate `Package@swift-5.6.swift` file that gets used when compiling with Xcode 5.6+.